packages/calcite-components/src/components/table-row/table-row.tsx

```diff
@@ -0,0 +1,353 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+  VNode,
+  Watch,
+} from "@stencil/core";
+import { LocalizedComponent } from "../../utils/locale";
+import { Scale, SelectionMode } from "../interfaces";
+import { focusElementInGroup, FocusElementInGroupDestination } from "../../utils/dom";
+import { RowType, TableRowFocusEvent } from "../table/interfaces";
+
+/**
+ * @slot - A slot for adding `calcite-table-cell` or `calcite-table-header` elements.
+ */
+
+@Component({
+  tag: "calcite-table-row",
+  styleUrl: "table-row.scss",
+  shadow: true,
+})
+export class TableRow implements LocalizedComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
+  @Prop({ reflect: true }) disabled = false;
```

Comment:
Do we need to implement `InteractiveComponent` for any table-supporting component w/ a `disabled prop?

Reply:
Added 👍